src/routes/events/index.tsx

```
@@ -1,51 +1,82 @@
-  const { events } = Route.useLoaderData()
+  const { events } = Route.useLoaderData();
   const navigate = Route.useNavigate();
 
-  // console.log(events[0].start_time)
+  const [searchTerm, setSearchTerm] = useState('');
+  const [isFocused, setIsFocused] = useState(false);
+
+  const filteredEvents = events.filter(
+    (event) =>
+      event.name &&
+      event.name.toLowerCase().includes(searchTerm.toLowerCase())
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)